src/main/java/org/fcrepo/migration/validator/api/ObjectValidationReport.java

```diff
@@ -26,21 +26,48 @@
  */
 public class ObjectValidationReport {
 
-    private String objectId;
+    /*
+     * Validation result which is the subject of this report
+     */
+    private ValidationResult result;
 
     /**
      * Constructor
-     * @param objectId of the object which is the subject of this report
+     * @param result of the object which is the subject of this report
      */
-    public ObjectValidationReport(final String objectId) {
-        this.objectId = objectId;
+    public ObjectValidationReport(final ValidationResult result) {
```

Comment:
@awoods :  I was envisioning here that the ObjectValidationReport would wrap a list of ValidationResults all of which relate to a single object.  So the validation results would all live under the object's validation json output subdirectory.